openvpn/process.go

```diff
@@ -102,9 +108,10 @@ func (process *Process) waitForShutdown(cmd *exec.Cmd) {
 	select {
 	// Wait for shutdown
 	case <-process.cmdShutdownStarted:
-		// Kill the server
-		if err := cmd.Process.Kill(); err != nil {
-			return
+		//First - shutdown gracefully
+		//TODO - add timer and send SIGKILL after timeout?
```

Comment:
This TODO is more about killing process immediately after a delay (in case graceful shutdown fails), and yes send(SIGKILL) and Kill() are equivalent. I think it could be added with separate PR

Reply:
Is it hard to implement delayed *force kill*?
At least, lets create ticket, or we will forget it soon

Reply:
Ticket was created.